<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenIGTLinkMeasurementsControls</class>
 <widget class="QWidget" name="OpenIGTLinkMeasurementsControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>392</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Start Measurement</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exportButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Export Measurement</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Max Measurement Points (0 = infinite)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxPointsSpinBox">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>285</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
